### PR TITLE
make boolean options have the same value

### PR DESCRIPTION
**Pull request by nyxnor**
https://github.com/rootzoll/raspiblitz/issues/2852

### CODE REVIEW

File: build_sdcard.sh [hunk 1]
```diff
@@ -116,9 +120,11 @@ range_argument(){
   fi
 }
 
-echo -e "\n*** SOFTWARE UPDATE ***"
 general_utils="curl"
-sudo apt install -y ${general_utils}
+if ! command -v curl >/dev/null; then
```
Comment (openoms) [inline build_sdcard.sh]: thanks, this would have been my other suggestion, to check for curl

**Comment by openoms** [comment]
LGTM

File: build_sdcard.sh (hunk 1 above)
Comment (nyxnor) [reply build_sdcard.sh]: improved now if there are multiple programs on general utils https://github.com/rootzoll/raspiblitz/pull/2912/commits/a0f7674c426818766174a64feb918bc9c71e6815

**Comment by openoms** [comment]
since curl is the only thing needed why not just keep it simple and get rid of the `general_utils` variable?

**Comment by nyxnor** [comment]
Just in case for the future if any utility is needed to be downloaded before confirmation.

**Comment by openoms** [comment]
consider readibility being a helpful attribute to help new contributors and who are using these scripts as a guide

**Comment by nyxnor** [comment]
I can add comments...

If command is not found on path, than add it to the list of commands to be installed
```sh
! command -v ${prog} >/dev/null && general_utils_install="${general_utils_install} ${prog##*/}"
```

If any program is not installed, install only the required ones
```sh
if [ -n "${general_utils_install}" ]; then
```
this avoids trying to install program if update fails
```sh
sudo apt update -y && sudo apt install -y ${general_utils_install}
```

**Comment by rootzoll** [comment]
I would like to make a new release candidate soon. Please signal me, when this is ready to merge.

**Comment by nyxnor** [comment]
on my part it is done

**Pull request merged**